style.css

```diff
@@ -0,0 +1,25 @@
+@media screen and (min-width: 650px) {
+    
+
+}
```

Comment:
Kindly remove the media query if you do not need it for future use. Or add a comment to explain it. This will help someone who may need to refer to your project.